mitosheet/setup.py

```diff
@@ -209,10 +209,10 @@
         packages                 = setuptools.find_packages(exclude=['deployment']),
         install_requires=[        
             "jupyterlab~=3.0",
-            'ipywidgets>=7,<9',
+            'ipywidgets>=8,<9',
             # In JLab 3, we move to needing to install the jupyterlab-widgets package, which
             # is equivlaent to the @jupyter-widgets/jupyterlab-manager extension for jlab 2
-            'jupyterlab-widgets>=1.0.0',
+            'jupyterlab-widgets>=3.0.0',
```

Comment:
I think we just want >= 3. Otherwise, I think the resolution looks for a version with 3.0.x.